CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unversioned
 
+- Major: New split for channels going live! /live
+
```

Comment:
Please add a changelog entry for the UX fix with tabbing in the "Select channel" dialog